using-mobile-extensions/adobe-experience-platform-messaging/README.md

```diff
@@ -0,0 +1,211 @@
+# Adobe Experience Platform Messaging
+
+The Adobe Experience Platform Messaging mobile extension allows you to send push notification tokens and push notification click-through feedback to the Adobe Experience Platform.
+
+{% hint style="warning" %}
+The Adobe Experience Platform Messaging extension is currently in Beta development.
+{% endhint %}
+
+## Prerequisites
+
+* IMS organization is enabled for messaging.
+* [Mobile Core](../mobile-core/README.md) and [Adobe Experience Platform Edge Network](../adobe-edge/README.md) extensions have been implemented in the app.
+
+## Configure Profile Dataset in Edge Configuration
+1. Follow the [edge configuration](../../getting-started/edge-configuration.md) document to configure edge.
+2. In the screen below update the **Profile Dataset** with the `CJM Push Profile Dataset`
+![Configuring your Edge Configuration in Adobe Experience Platform Launch](../../.gitbook/assets/screen-shot-2021-02-02-at-12.44.31-pm.png)
+3. Hit **Save**.
+
+## Configure the Adobe Experience Platform Messaging extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Adobe Experience Platform - Messaging** extension, and click **Install**.
+3. Select the **Event Dataset** for Production, Stage and Development Environments. The datasets selected should use a schema that uses the Push Notification Tracking mixin. For more information, see [Setup Schemas & Datasets](../../getting-started/configure-schema-and-dataset.md).
```

Comment:
i wonder if we need more information around the tracking mixin.  what do you think?

Reply:
I think we should have this somewhere else and we should link to that.